cypress/schemas/liveblog-schema.js

```diff
@@ -0,0 +1,72 @@
+import { versionSchemas } from '@cypress/schema-tools';
+import { blog100 } from './blog-schema';
+
+const liveblog100 = {
+  ...blog100,
+  schema: {
+    ...blog100.schema,
+    title: 'LiveBlogPosting',
+    description:
+      'An example schema describing JSON-LD for type: LiveBlogPosting',
+    properties: {
+      ...blog100.schema.properties,
+      coverageStartTime: {
+        type: 'string',
+        description:
+          'The time when the live blog will begin covering the Event. ' +
+          "Note that coverage may begin before the Event's start time. The LiveBlogPosting may also be created before coverage begins.",
+      },
+      coverageEndTime: {
+        type: 'string',
+        description:
+          'The time when the live blog will stop covering the Event. Note that coverage may continue after the Event concludes.',
+      },
+      liveBlogUpdate: {
+        type: 'array',
+        items: {
+          ...blog100.schema,
+          required: ['@type'],
```

Comment:
Not all the properties of `BlogPosting` are required, where could I find out which props are really required in this context ?